TwitterHashtagScrape.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import tweepy
+import json
+from tweepy import api
+
+access_token="" #Add your token
+access_token_secret="" #Add your access_token_secret token
+
+api_key="" #Add your api
+api_key_secret="" #Add your api_key_secret token
+
+auth = tweepy.OAuthHandler(consumer_key=api_key,consumer_secret=api_key_secret)
+auth.set_access_token(access_token,access_token_secret)
+
+api = tweepy.API(auth)
+print
+
+#def write_Json(data, filename="hashtag.json"):
+#    with open(filename,"w") as f:
+#        json.dump(data,f,indent=4)
+
+india_woeid=23424848 # Where On Earth IDentifier
+
+trend_result=api.trends_place(india_woeid)
+
+try:
+    for trend in trend_result[0]["trends"][:10]:
+        print(trend["name"]+ " : "+str(trend["tweet_volume"]))
```

Comment:
It's only for reading data from tweepy library right? You cannot add data to it

Reply:
This program will fetch the top 50 hashtags and print, besides the data which I fetched was JSON data so, instead of passing it again to a JSON file I just printed it out?